features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/HostResourceSwRunMonitor.java

```diff
@@ -198,7 +197,7 @@ public void rowCompleted(SnmpRowResult result) {
                 final SnmpValue name = nameResults.get(nameInstance);
                 final SnmpValue value = statusResults.get(nameInstance);
                 // See if the service name is in the list of running services
-                if (name != null && value != null && match(serviceName, stripExtraQuotes(name.toString()))) {
+                if (name != null && value != null && match(serviceName, StringUtils.stripExtraQuotes(name.toString()))) {
```

Comment:
The only difference is that antlr.StringUtils.stripFrontBack() throws a StringIndexOutOfBoundsException if you pass it ```"```, and we were only handling this in one of the methods.